controllers/glance_controller.go

```diff
@@ -60,10 +60,14 @@ import (
 type GlanceReconciler struct {
 	client.Client
 	Kclient kubernetes.Interface
-	Log     logr.Logger
 	Scheme  *runtime.Scheme
 }
 
+// GetLogger returns a logger object with a logging prefix of "controller.name" and additional controller context fields
```

Comment:
Wondering if we can move the `GetLogger` function in `glance_common.go` and eventually pass (as input) the `component`.
This way we can extend the behavior and we don't have to define it twice, as well as we can make it independent from the caller. @pinikomarov can you check this?

Reply:
Basically add:

```
// GetLogger returns a logger object with a logging prefix of "controller.name" and additional controller context fields
func GetLogger(ctx context.Context, caller string) logr.Logger {
	return log.FromContext(ctx).WithName("Controllers").WithName(caller)
}
```
to `glance_common.go`, and in both `glance_controller.go` and `glanceapi_controller.go` change the `GetLogger` call to something like:

```golang
// Reconcile reconcile Glance API requests
func (r *GlanceAPIReconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ctrl.Result, _err error) {
	Log := GetLogger(ctx, string(glance.GlanceAPI))
...
...
```

and

```golang
// Reconcile reconcile Glance requests
func (r *GlanceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ctrl.Result, _err error) {
	Log := GetLogger(ctx, string(glance.Glance))
...
...
```